lint.sh

```diff
@@ -0,0 +1,7 @@
+echo "Linting!"
```

Comment:
What advantage does this offer over running the normal tox task and should we move this into a `scripts` folder?